### PR TITLE
Add email with discord details

### PR DESCRIPTION
**Pull request by RealOrangeOne**
This email will be sent to each team, with the relevant details added.

### CODE REVIEW

File: SR2022/2021-11-12-discord-credentials.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+to: Student Robotics 2022 teams
+subject: Student Robotics Discord Details
+---
+
+Hi
+
+This year, we're using [Discord](https://studentrobotics.org/docs/team_admin/discord) as our primary communication method for competitor support. Your invite link is [discord link]().
+
+Upon joining the server users will need to provide a password to gain the correct permissions. For your team, the password is: `password`. Please share both the invite link and the relevant password with your competitors. Discord access should not be shared to those outside your team.
```
Comment (PeterJCLaw) [inline SR2022/2021-11-12-discord-credentials.md:10]: Is it worth mentioning here that each team has their own channel, so this password is important to ensure that people end up in the right channel?

File: SR2022/2021-11-12-discord-credentials.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+to: Student Robotics 2022 teams
+subject: Student Robotics Discord Details
```
Comment (Adimote) [inline SR2022/2021-11-12-discord-credentials.md:3]:
```suggestion
subject: Student Robotics Discord Forum Details
```

File: SR2022/2021-11-12-discord-credentials.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Upon joining the server users will need to provide a password to gain the correct permissions. The password is used to identify which team a user is from, so do not share to those outside your team. For your team, the password is: `password`. Please share both the invite link and the relevant password with your competitors.
+
+If competitors change teams or need their roles tweaked for whatever reason please ask them to ask for help in the [#role-requests](https://discord.com/channels/900501415548579842/900501416269971460) channel.
+
+More information on this can be found in [the docs](https://studentrobotics.org/docs/team_admin/discord).
+
+Additionally, we like to make sure we are aware which members are blueshirts, competitors and team supervisors. To get the 'Team Supervisor' role on Discord please reply to this email with your Discord username once you have joined the server and we will add the appropriate roles.
```
Comment (Adimote) [inline SR2022/2021-11-12-discord-credentials.md:16]:
```suggestion
Additionally, we like to make sure we are aware which members are volunteers, competitors and team supervisors. To get the 'Team Supervisor' role on Discord please reply to this email with your Discord username once you have joined the server and we will add the appropriate roles.
```

**Review by Adimote — APPROVED** [review]
LGTM some small comments which you can take if you want

**Pull request merged**